More2Life.xcodeproj/project.pbxproj

```diff
@@ -991,8 +1035,8 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = com.coachkalani.More2Life;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE = "a5a22912-8745-4c07-87e1-461f297c4457";
-				PROVISIONING_PROFILE_SPECIFIER = "match AppStore com.coachkalani.More2Life";
```

Comment:
I think i had this setup right before...

Reply:
oh i see. match lets you choose `development, adhoc, enterprise  or appstore` and you are using the appstore profile. fixed.